TradeItIosTicketSDK2/TradeItAccountSelectionViewController.swift

```diff
@@ -37,6 +37,12 @@ class TradeItAccountSelectionViewController: UIViewController, TradeItAccountSel
                     onAnswerSecurityQuestion: answerSecurityQuestion
                 )
             },
+            onFailure:  { (tradeItErrorResult: TradeItErrorResult, linkedBroker: TradeItLinkedBroker) -> Void in
```

Comment:
Not sure the best way to do that, but it works (except for multiple alerts popup, we need to do the story #131883563).